### DISCUSSION TITLE
Hey Henry, I want to contribute to js draw and joplin

### DISCUSSION THREAD

**Discussion by Wladefant** [personalizedrefrigerator/js-draw#41] [Q&A]
Hey Henry, 
I would like to contribute to joplin and js draw, because I use it myself and I think that would be a perfect experience and also I want to bring in some design consistency into the app, in that regard Obsidian is for sure better. 

I will start studying at the university of munich next week. Are you still studying? Either way, I would like to have your contacts (twitter, discord) so we can contact a bit and you can help me when i dont understand something in the codebase if that is alright with you. I would also happily contribute to js draw as that would be also very interesting. 

I vow to never leave the two projects IF I learn the codebase 🌝 

In the future we could do some projects together (preferably for joplin). I would like to work on the mobile app in the next year or so. Hope to hear from you soon.

**Comment by personalizedrefrigerator** [comment]
Hi,

If you're interested in working on a particular feature, I would be happy to direct you to the relevant parts of the codebase (either for js-draw or Joplin). For `js-draw`, [CONTRIBUTING.md](https://github.com/personalizedrefrigerator/js-draw/blob/main/CONTRIBUTING.md) should be helpful (but it might be somewhat outdated).

I don't have Twitter, but my Discord username is personalizedrefrigerator.

(I graduated in June and am now doing work for the Joplin company.)

**Reply by Wladefant** [reply]
Oh for real. I just thought you are a contributor. Nice. So how many paid employees does joplin has. You and laurent or someone else also?

**Reply by Wladefant** [reply]
Right now I would like to learn to learn everything about react native and improving the mobile app for the next year. To get something like the obsidian mobile app, because right now our mobile app is just an embarassment. So that it looks at least like the desktop app for tablets. I will write you on discord then ;)

**Reply by Wladefant** [reply]
But the google summer of code was from goodwill right?

**Reply by personalizedrefrigerator** [reply]
GSoC was paid (but work between September and June was unpaid -- I was using js-draw & Joplin to take notes for my classes :) ).

Sorry for taking 5 days to reply!!!
